### ISSUE TITLE
Custom badges for docs and benchmark

### ISSUE THREAD

**Issue by Zitrone44** [thm-mni-ii/graph-algo-ptas#55]
Currently the README badges for docs and benchmark look the same and can only be distinguished using the alt text. This should be changed.
